templates/taxi/manufacturer_list.html

```diff
@@ -2,31 +2,33 @@
 
 {% block content %}
     <h1>Manufacturer List
+      <a href="{% url 'taxi:manufacturer-create' %}" style="color: blue"> + </a>
     </h1>
 
     {% if manufacturer_list %}
-      <table class="table">
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
-        </tr>
-
-        {% for manufacturer in manufacturer_list %}
-            <tr>
-              <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
-              </td>
+      <table class="table table-striped">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Country</th>
+            <th>Actions</th>
           </tr>
-        {% endfor %}
+        </thead>
+        <tbody>
+          {% for manufacturer in manufacturer_list %}
+              <tr>
+                <td>{{ manufacturer.id }}</td>
+                <td>{{ manufacturer.name }}</td>
+                <td>{{ manufacturer.country }}</td>
+                <td>
+                  <a href="{% url 'taxi:manufacturer-update' manufacturer.id %}" class="btn btn-primary btn-sm">Update</a>
+                  <a href="{% url 'taxi:manufacturer-delete' manufacturer.id %}" class="btn btn-danger btn-sm">Delete</a>
```

Comment:
  The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.